commit dd0460697c4e281fe277b53916d5251332e1aafc
Author: Jiří Klimeš <jklimes@redhat.com>
Date:   Tue Mar 27 22:36:55 2012 +0200

    libnm-glib: initialize NMRemoteSettings in nm_remote_settings_new() (rh #806664)
    
    The object was not initialized after creation in nm_remote_settings_new(). This
    was a regression caused by 762df85234e7a042a2a5d31053e6cc273ae3e2ec.

diff --git a/libnm-glib/nm-remote-settings.c b/libnm-glib/nm-remote-settings.c
index 8f213f9..b6b6c18 100644
--- a/libnm-glib/nm-remote-settings.c
+++ b/libnm-glib/nm-remote-settings.c
@@ -728,7 +728,12 @@ properties_changed_cb (DBusGProxy *proxy,
 NMRemoteSettings *
 nm_remote_settings_new (DBusGConnection *bus)
 {
-	return g_object_new (NM_TYPE_REMOTE_SETTINGS, NM_REMOTE_SETTINGS_BUS, bus, NULL);
+	NMRemoteSettings *settings;
+
+	settings = g_object_new (NM_TYPE_REMOTE_SETTINGS, NM_REMOTE_SETTINGS_BUS, bus, NULL);
+
+	_nm_remote_settings_ensure_inited (settings);
+	return settings;
 }
 
 static void
